src/util/VanDerCorput.cpp

```diff
@@ -116,10 +131,8 @@ VanDerCorput::const_iterator VanDerCorput::end() const
 }
 
 //=============================================================================
-VanDerCorput::const_iterator::const_iterator(const VanDerCorput *seq)
-    : mSeq(seq)
-    , mN(-1)
-    , mFinalIter(false)
+VanDerCorput::const_iterator::const_iterator(const VanDerCorput* seq)
+  : mSeq(seq), mN(-1), mFinalIter(false)
```

Comment:
I _really_ don't like putting an initialization list on one line. I don't see the difference between this initialization list:
```c++
: mSeq(seq), mN(-1), mFinalIter(false)
```
and this list of assignments:
```c++
mSeq = seq; mN = -1; mFinalIter = false;
```
We wouldn't allow the latter, so why allow the former? 😉 

---

On a slight tangent, I prefer:
```c++
: mSeq(seq)
, mN(-1)
, mFinalIter(false)
```
over
```c++
: mSeq(seq),
  mN(-1),
  mFinalIter(false)
```
because it's easier to re-order lines. I think @psigen strongly prefers the latter, though. 😞